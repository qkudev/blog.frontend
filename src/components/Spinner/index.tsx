import * as React from 'react'
import './style'

function Spinner() {
  return (
    <div className='lds-facebook'>
      <div />
      <div />
      <div />
    </div>
  )
}

export default Spinner
