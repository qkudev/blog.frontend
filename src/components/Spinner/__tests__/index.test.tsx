import * as React from 'react'
import renderer from 'react-test-renderer'
import Spinner from '../index'

describe('Spinner', function() {
  test('Matches with snapshot', function() {
    const header = renderer.create(<Spinner />)
    let tree = header.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
