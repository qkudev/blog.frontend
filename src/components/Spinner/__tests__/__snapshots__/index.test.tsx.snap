// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Matches with snapshot 1`] = `
<div
  className="lds-facebook"
>
  <div />
  <div />
  <div />
</div>
`;
